Library/Formula/synergy.rb

```diff
@@ -0,0 +1,75 @@
+class Synergy < Formula
+  homepage 'http://synergy-project.org'
+  url 'https://github.com/synergy/synergy/archive/1.6.3.tar.gz'
+  sha1 'c4c26ee983f0684a5cbe45520adc0b50a71a3524'
+
+  depends_on "cmake" => :build
+  depends_on "qt5" => :build
+
+  def install
+    ENV.deparallelize  # if your formula fails when building in parallel
+
+    system './hm.sh', 'conf', '-g1', '--mac-sdk', '10.10', '--mac-identity', 'test'
```

Comment:
Heh. I forgot it'd get punchy about that. I presume it wants you to change it to:

```ruby
system "./hm.sh", "conf", "-g1", "--mac-sdk", MacOS.version, "--mac-identity", "test"
```